### PR TITLE
Command regrouping to remove verify command from ascmhl tool

### PR DESCRIPTION
**Pull request by ptrpfn**
- introduced `ascmhl-debug` tool (now with verify, xsd-schema-check, hash commands)
- removing `verify` command from `ascmhl` tool
- adapting README

### CODE REVIEW

File: ascmhl/generator.py [hunk 1]
```diff
@@ -41,6 +41,78 @@ def __init__(self, history: MHLHistory, ignore_spec: MHLIgnoreSpec = MHLIgnoreSp
...
+        # for collections we cannot create a valid relative path (we are in the "wrong" history), but in that case
+        # the file_path is inputted already as the relative path (a bit of implicit functionality here)
+        if history_relative_path == None:
+            history_relative_path = file_path
+
+        # check if there is an existing hash in the other generations and verify
+        original_hash_entry = history.find_original_hash_entry_for_path(history_relative_path)
+
+        hash_entries = [MHLHashEntry]
+        # TODO: sort the format keys into a standard order for consistent output
```
Comment (mthudgins) [inline ascmhl/generator.py:71]: I think this TODO can be eliminated

File: tests/test_create.py [hunk 2]
```diff
@@ -310,3 +325,39 @@ def test_creator_info(fs, simple_mhl_history):
...
+    runner = CliRunner()
+    result = runner.invoke(ascmhl.commands.create, ["/root", "-v", "-h", "md5", "-h", "sha1"])
+    assert result.exit_code == 0
+
+    assert "A/A1.txt  md5: fe6975a937016c20b43b17540e6c6246" in result.output
+    assert "A/A1.txt  sha1: 4a5b95edbea7de5ed2367432645df88cd4f1d1b6" in result.output
+
+
+@freeze_time("2020-01-16 09:15:00")
+def test_create_mulitple_hashformats_double_hashformat(fs, simple_mhl_history):
```
Comment (mthudgins) [inline tests/test_create.py:349]: This test is still failing for me locally.  Is this expected behavior?

File: ascmhl/commands.py [hunk 3]
```diff
@@ -43,12 +45,13 @@
     is_flag=True,
     help="Verbose output",
 )
+# FIXME: refactor to allow for multiple hash formats
```
Comment (mthudgins) [inline ascmhl/commands.py:48]: This FIXME can probably be removed

File: ascmhl/commands.py [hunk 4]
```diff
@@ -376,6 +424,7 @@ def create_for_single_files_subcommand(
 )
 # options
 @click.option(
+    # FIXME: Update to permit multiple hash formats
```
Comment (mthudgins) [inline ascmhl/commands.py:427]: FIXME can probably be removed

**Review by mthudgins — COMMENTED** [review]
Left a couple of comments but noticed nothing critical other than test_create_mulitple_hashformats_double_hashformat still being broken.  I think this was a good restructure.  I'm still looking through the changes in the README but didn't want to delay this feedback

File: ascmhl/generator.py (hunk 1 above)
Comment (ptrpfn) [reply ascmhl/generator.py:71]: It looks to me as if the method `append_multiple_format_file_hashes` is currently not used. Did you add that temporarily?

File: tests/test_create.py (hunk 2 above)
Comment (ptrpfn) [reply tests/test_create.py:349]: I think `pytest .` actually runs the test successfully (e.g. added log output to that test that was printed). But I also experienced the test failing from within PyCharm – but also with a wrong line number of the assert. So I would trust the pytest command and the test on github for now that it runs successfully.

**Comment by ptrpfn** [comment]
@mthudgins Can you take a look at the test again? If it works for you with `pytest .` from the repository's root directory, you could approve the PR.

File: ascmhl/generator.py (hunk 1 above)
Comment (mthudgins) [reply ascmhl/generator.py:71]: I believe I created this function as a way to streamline appending of file hash values to the session, however the current implementation did not lend itself to utilizing this as in certain cases it is only desirable to append a subsection of the hashes generated.  There may be a use case for this in the future, but the way things are currently implemented there is not without some refactor work.  I'll leave it to you whether to keep this function or remove it

File: tests/test_create.py (hunk 2 above)
Comment (mthudgins) [reply tests/test_create.py:349]: You are correct.  running pytest . completes successfully.  Appears to be a problem with Pycharm

**Review by mthudgins — APPROVED** [review]

**Pull request merged**